config/default/app_settings.json

```diff
@@ -264,7 +264,8 @@
     ],
     "can_upgrade": [
       "program_officer"
-    ]
+    ],
+    "can_view_old_filter_and_search": []
```

Comment:
Nice! That makes it easy to play with this feature flag locally

Reply:
Thanks!